src/System.Text.Json/ref/System.Text.Json.cs

```diff
@@ -114,6 +117,8 @@ public sealed partial class JsonDocument : System.IDisposable
         public static System.Text.Json.JsonDocument Parse(System.ReadOnlyMemory<byte> utf8Json, System.Text.Json.JsonDocumentOptions options = default(System.Text.Json.JsonDocumentOptions)) { throw null; }
         public static System.Text.Json.JsonDocument Parse(System.ReadOnlyMemory<char> json, System.Text.Json.JsonDocumentOptions options = default(System.Text.Json.JsonDocumentOptions)) { throw null; }
         public static System.Text.Json.JsonDocument Parse(string json, System.Text.Json.JsonDocumentOptions options = default(System.Text.Json.JsonDocumentOptions)) { throw null; }
+        public static System.Text.Json.JsonDocument Parse(System.Text.Json.JsonElement node) { throw null; }
+        public static System.Text.Json.JsonDocument Parse(System.Text.Json.JsonNode node) { throw null; }
```

Comment:
What's the scenario here?  `Parse` is the wrong verb for either of these, because it means "read text and produce a value".  These methods wouldn't be reading text, so if there's a need to produce an immutable document from a JsonNode it should perhaps live on Node as an export-type method, not on Document as an import-type method.

Reply:
I used `Parse(JsonNode)`, because I wanted to underline that this function will really go through the whole structure, not just box it like `AsJsonElement`. I thought `Parse(JsonNode)` would behave as  `Parse(JsonNode)` if there's a JsonNode underneath and return reference to JsonDocument it hold if not. 

But I can change them to for example `JsonNode.ToImmutable()` and `JsonElement.ToImmutable()` , which would show their main functionality?